test/integration/webhook/resourceflavor_test.go

```diff
@@ -69,18 +85,58 @@ var _ = ginkgo.Describe("ResourceFlavor Webhook", func() {
 		})
 	})
 
-	ginkgo.When("Creating a ResourceFlavor with invalid taints", func() {
-		ginkgo.It("Should fail to create", func() {
-			ginkgo.By("Creating a new resourceFlavor")
-			resourceFlavor := testing.MakeResourceFlavor("resource-flavor").Taint(corev1.Taint{
-				Key:    "@foo",
-				Value:  "bar",
-				Effect: corev1.TaintEffectNoSchedule,
-			}).Obj()
+	ginkgo.When("Creating a ResourceFlavor with invalid values", func() {
```

Comment:
Maybe you can use a test table instead?
Similar to this https://github.com/kubernetes-sigs/kueue/blob/34bfbe0f359b8439b737f07c3f3e5da92c7d0d67/test/integration/webhook/clusterqueue_test.go#L158